File: CloudFlow/FrontEnd-App/src/app/shared/model/curs-valutar/curs-valutar.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class CursValutarModel {

	@JsonProperty("id", Number)
	public id: number = null;

	@JsonProperty("eur", Number)
	public eur: number = null;

	@JsonProperty("usd", Number)
	public usd: number = null;

	@JsonProperty("data", JsonDateConverter)
	public data: Date = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/cursValutar/CursValutarModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.cursValutar;

import java.math.BigDecimal;
import java.util.Date;

public class CursValutarModel {

	private Long id;
	private BigDecimal eur;
	private BigDecimal usd;
	private Date data;
	
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public BigDecimal getEur() {
		return eur;
	}
	public void setEur(BigDecimal eur) {
		this.eur = eur;
	}
	public BigDecimal getUsd() {
		return usd;
	}
	public void setUsd(BigDecimal usd) {
		this.usd = usd;
	}
	public Date getData() {
		return data;
	}
	public void setData(Date data) {
		this.data = data;
	}
}
